{"uid":"e3558c5fabb201c3","name":"Forgot Password with valid email test","fullName":"tests.forgot_password_test.TestForgotPassword#test_valid_email","historyId":"d6dde1715fb973c93e423cd6a34e3dff","time":{"start":1748483579797,"stop":1748483579797,"duration":0},"description":"Forgot password with a valid email address","descriptionHtml":"<p>Forgot password with a valid email address</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1748483587467,"stop":1748483587467,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1748483587467,"stop":1748483587468,"duration":1},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"base_url","time":{"start":1748483587467,"stop":1748483587467,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1748483587467,"stop":1748483587467,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1748483612112,"stop":1748483612112,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Forgot password with a valid email address","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","steps":[{"name":"public ip address","time":{"start":1748483587583,"stop":1748483587595,"duration":12},"status":"passed","steps":[],"attachments":[{"uid":"de025c8a380897ef","name":"HTTP logs of http://checkip.amazonaws.com/","source":"de025c8a380897ef.txt","type":"text/plain","size":357}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"a3de23a0c9db1fad","name":"Cookies","source":"a3de23a0c9db1fad.json","type":"application/json","size":4118},{"uid":"b2e4fe4aa34a7c94","name":"Session Storage","source":"b2e4fe4aa34a7c94.json","type":"application/json","size":359},{"uid":"75d746df768decf8","name":"Local Storage","source":"75d746df768decf8.json","type":"application/json","size":548},{"uid":"bcfbe2aca810dd30","name":"Full Page Screenshot","source":"bcfbe2aca810dd30.png","type":"image/png","size":1363424},{"uid":"e86d13347ea0fe2c","name":"URL","source":"e86d13347ea0fe2c.uri","type":"text/uri-list","size":23},{"uid":"ac14f6bab7c612e5","name":"Network Logs","source":"ac14f6bab7c612e5.json","type":"application/json","size":48956}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":7,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1748483618497,"stop":1748483618498,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"subSuite","value":"Custom sub suite"},{"name":"story","value":"Forgot Password Feature's Functionality"},{"name":"severity","value":"critical"},{"name":"suite","value":"Custom suite"},{"name":"epic","value":"Security"},{"name":"parentSuite","value":"Custom parent suite"},{"name":"tag","value":"security"},{"name":"host","value":"fv-az1068-680"},{"name":"thread","value":"2059-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.forgot_password_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":20,"skipped":0,"passed":0,"unknown":0,"total":20},"items":[{"uid":"e53dec4e92921ef4","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/667//#testresult/e53dec4e92921ef4","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748397198175,"stop":1748397198175,"duration":0}},{"uid":"38c3b9fe248e4b55","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/666//#testresult/38c3b9fe248e4b55","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748310668239,"stop":1748310668239,"duration":0}},{"uid":"fce4747def74ac71","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/665//#testresult/fce4747def74ac71","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748224535823,"stop":1748224535823,"duration":0}},{"uid":"56a9f4f8f69fe58e","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/664//#testresult/56a9f4f8f69fe58e","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748138448981,"stop":1748138448981,"duration":0}},{"uid":"53929c8bc0a70fe7","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/663//#testresult/53929c8bc0a70fe7","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748051278238,"stop":1748051278238,"duration":0}},{"uid":"ac52ff457b97636","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/662//#testresult/ac52ff457b97636","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747965122052,"stop":1747965122052,"duration":0}},{"uid":"fbcd5e60960642e1","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/661//#testresult/fbcd5e60960642e1","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747878726284,"stop":1747878726284,"duration":0}},{"uid":"d89766d733395c9","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/660//#testresult/d89766d733395c9","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747792364522,"stop":1747792364522,"duration":0}},{"uid":"9ddfa701c6ad50e","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/659//#testresult/9ddfa701c6ad50e","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747705990466,"stop":1747705990466,"duration":0}},{"uid":"e879b2dcd734d3ec","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/658//#testresult/e879b2dcd734d3ec","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747619822833,"stop":1747619822833,"duration":0}},{"uid":"ff653d525d571c1","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/657//#testresult/ff653d525d571c1","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747533485624,"stop":1747533485624,"duration":0}},{"uid":"6d9efdc24454c450","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/656//#testresult/6d9efdc24454c450","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747446545681,"stop":1747446545681,"duration":0}},{"uid":"61bb60eaf6e018c","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/655//#testresult/61bb60eaf6e018c","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747360362790,"stop":1747360362790,"duration":0}},{"uid":"628e9e4706c797ce","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/654//#testresult/628e9e4706c797ce","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747273759299,"stop":1747273759299,"duration":0}},{"uid":"9f78bb0b0ec4285f","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/653//#testresult/9f78bb0b0ec4285f","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747187467021,"stop":1747187467021,"duration":0}},{"uid":"ceb1265741529239","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/652//#testresult/ceb1265741529239","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747101147402,"stop":1747101147402,"duration":0}},{"uid":"290dfe77ac2a258b","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/651//#testresult/290dfe77ac2a258b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747014901406,"stop":1747014901406,"duration":0}},{"uid":"d1e41bac06e661e","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/650//#testresult/d1e41bac06e661e","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1746928594663,"stop":1746928594663,"duration":0}},{"uid":"365ee8d6633dbc09","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/649//#testresult/365ee8d6633dbc09","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1746841582414,"stop":1746841582414,"duration":0}}]},"tags":["security"]},"source":"e3558c5fabb201c3.json","parameterValues":[]}